src/server/assets/index.test.js

```diff
@@ -1,13 +1,25 @@
+/* eslint-disable global-require */
 import path from 'path';
-import getAssetsArray from '.';
+import nodeLogger from '../../app/helpers/logger.node';
+
+const mockLogError = jest.fn();
+
+jest.mock('../../app/helpers/logger.node', () => jest.fn());
+
+nodeLogger.mockImplementation(() => ({ error: mockLogError }));
 
 describe('getAssetsArray', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
   describe('no assets manifest', () => {
-    it('should console log an error', async () => {
+    it('should log an error', async () => {
       delete process.env.SIMORGH_ASSETS_MANIFEST_PATH;
-      global.console.log = jest.fn();
+      const getAssetsArray = require('./index.js').default;
```

Comment:
I might be missing some intentional behaviour, but why does this line repeat? Feels like we should be able to pull it out into one, & if we need to reset it or something we should be able to do that in a before-/after-Each?

Reply:
It needs to be after the line that changes SIMORGH_ASSETS_MANIFEST_PATH in each test as it runs on import/require